### PR TITLE
NAS-104317 

### PR DESCRIPTION
**Pull request by lola27**
portals; enable to let user select one if multiple admin portals
available

### CODE REVIEW

File: src/app/pages/plugins/plugins.component.ts [hunk 1]
```diff
@@ -432,7 +432,29 @@ export class PluginsComponent {
...
       onClick: (row) => {
-        window.open(row.admin_portal);
+        if (row.admin_portals.length > 1) {
+          const conf: DialogFormConfiguration = {
+            title: 'Go to admin portal',
+            fieldConfig: [
+              {
+                type: 'select',
+                name: 'admin_portal',
+                placeholder: 'Please select the admin portal',
```
Comment (erinix) [inline src/app/pages/plugins/plugins.component.ts]: please put in helptext file and add T() to translate

File: src/app/pages/plugins/plugins.component.ts [hunk 2]
```diff
@@ -432,7 +432,29 @@ export class PluginsComponent {
...
+                options: row.admin_portals ? row.admin_portals.map(item => {return {label: item, value: item}}) : [],
+                value: row.admin_portals[0]
+              }
+            ],
+            saveButtonText: 'Go to Portal',
```
Comment (erinix) [inline src/app/pages/plugins/plugins.component.ts]: Here too

File: src/app/pages/plugins/plugins.component.ts [hunk 3]
```diff
@@ -432,7 +432,29 @@ export class PluginsComponent {
       label: T("MANAGE"),
       icon: 'settings',
       onClick: (row) => {
-        window.open(row.admin_portal);
+        if (row.admin_portals.length > 1) {
+          const conf: DialogFormConfiguration = {
+            title: 'Go to admin portal',
```
Comment (erinix) [inline src/app/pages/plugins/plugins.component.ts]: and this one :)

**Comment by erinix** [comment]
@freenas/docs-team could you please review the text

File: src/app/helptext/plugins/plugins.ts [hunk 4]
```diff
@@ -47,4 +47,11 @@ multi_update_dialog: {
     content: T('Updating selected plugins.'),
     succeed: T('Selected plugins updated.'),
 },
+
+portal_dialog: {
+    title: T('Go to Admin Portal'),
+    admin_portal_placeholder: T('Select the admin portal'),
```
Comment (wblock) [inline src/app/helptext/plugins/plugins.ts]:
```suggestion
    admin_portal_placeholder: T('Select the admin portal:'),
```

**Pull request merged**